File: intetest/lab3_001.pas
program test(input,output);
const a=1; //整型常量
      b=3.0; //浮点型常量
      c='c'; //字符型常量
      blank=' '; //空格
var   d:integer; //整型变量
      e:real; //浮点型变量
      f:char; //字符型变量
      g:boolean; //布尔型变量
begin
    d:=6;
    e:=6.0;
    f:='f';
    g:=2>1;//不支持true常量，所以这么写
    writeln(a,blank,b,blank,c,blank,1,blank,1.0,blank,'y');
    write(a,blank,b,blank,c,blank,1,blank,1.0,blank,'y');
    writeln;
    writeln(g);
    writeln(a+b,blank,a*b,blank,g,blank,a>b,blank,c,blank,d,blank,e,blank,d<>e);
end.

File: intetest/lab3_001.c
#include<stdio.h>
#include<stdbool.h>
const int a = 1;
const double b = 3.0000000000000000;
const char c = 'c';
const char blank = ' ';
int d;
double e;
char f;
bool g;
int main() {
d = 6;
e = 6.0000000000000000;
f = 'f';
g = 2 > 1;
printf("\n");
printf("%d%c%.6lf%c%c%c%d%c%.6lf%c%c",a, blank, b, blank, c, blank, 1, blank, 1.0000000000000000, blank, 'y');
printf("\n");
printf("\n");
printf("\n");
return 0;
}
